L/Libxc/Libxc_GPU/build_tarballs.jl

```diff
@@ -5,42 +5,64 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "Libxc_GPU"
-version = v"6.1.0"
+version = v"7.0.0"
 include("../sources.jl")
 
-sources = [
-    sources;
-    DirectorySource("./bundled")
-]
-
 # Bash recipe for building GPU version
 # Notes:
 #   - 3rd and 4th derivatives (KXC, LXC) not built since gives a binary size of ~200MB
+#   - For compilation on aarch64, the official x86_86 CUDA redist is downloaded, in order
+#     to enable cross compilation with NVCC
 script = raw"""
 cd $WORKSPACE/srcdir/libxc-*/
 
-# Needed for Libxc 6.1.0 as these backport some fixes on libxc master
-# On Libxc > 6.1.0 we can also remove the -DBUILD_TESTING=OFF
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/cmake-cuda.patch
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/source-fixes.patch
+# CMake looks for CUDA libraries in lib64
+ln -s $prefix/cuda/lib $prefix/cuda/lib64
+
+# Necessary operations to cross compile CUDA from x86_64 to aarch64
+if [[ "${target}" == aarch64-linux-* ]]; then
+
+   # Add /usr/lib/csl-musl-x86_64 to LD_LIBRARY_PATH to be able to use host nvcc
+   export LD_LIBRARY_PATH="/usr/lib/csl-musl-x86_64:/usr/lib/csl-glibc-x86_64:/usr/local/lib64:/usr/local/lib:/usr/lib64:/usr/lib:/lib64:/lib:/workspace/x86_64-linux-musl-cxx11/destdir/lib:/workspace/x86_64-linux-musl-cxx11/destdir/lib64:/opt/x86_64-linux-musl/x86_64-linux-musl/lib64:/opt/x86_64-linux-musl/x86_64-linux-musl/lib:/opt/${target}/${target}/lib64:/opt/${target}/${target}/lib:/workspace/destdir/lib64"
```

Comment:
So, in the Reactant recipe I went with the super explicit path because bazel is terrible and handling env vars is complicated, I think this can be simplified here with
```suggestion
   export LD_LIBRARY_PATH="/usr/lib/csl-musl-x86_64:/usr/lib/csl-glibc-x86_64:${LD_LIBRARY_PATH}"
```

Reply:
The simplification works, and it looks much cleaner. Thanks